draft/2022-11-23-this-week-in-rust.md

```diff
@@ -77,7 +77,26 @@ If you are a Rust project owner and are looking for contributors, please submit
 
 ### Rust Compiler Performance Triage
 
-<!-- Perf results go here -->
+A fairly quiet week with regressions unfortunately slightly outweighing regressions. There was not any particular change of much note. Many of the regressions were justifiable since they were for critical bug fixes.
```

Comment:
```suggestion
A fairly quiet week with regressions unfortunately slightly outweighing improvements. There was not any particular change of much note. Many of the regressions were justifiable since they were for critical bug fixes.
```